lib/database.g.dart

```diff
@@ -0,0 +1,222 @@
+      'stop': serializer.toJson<DateTime?>(stop),
+    };
+  }
+
+  Timer copyWith(
+          {int? id,
+          DateTime? start,
+          Value<DateTime?> stop = const Value.absent()}) =>
+      Timer(
+        id: id ?? this.id,
+        start: start ?? this.start,
+        stop: stop.present ? stop.value : this.stop,
+      );
+  @override
+  String toString() {
+    return (StringBuffer('Timer(')
+          ..write('id: $id, ')
+          ..write('start: $start, ')
+          ..write('stop: $stop')
+          ..write(')'))
+        .toString();
+  }
+
+  @override
+  int get hashCode => Object.hash(id, start, stop);
+  @override
+  bool operator ==(Object other) =>
+      identical(this, other) ||
+      (other is Timer &&
+          other.id == this.id &&
+          other.start == this.start &&
+          other.stop == this.stop);
+}
+
+class TimersCompanion extends UpdateCompanion<Timer> {
+  final Value<int> id;
+  final Value<DateTime> start;
+  final Value<DateTime?> stop;
+  const TimersCompanion({
+    this.id = const Value.absent(),
+    this.start = const Value.absent(),
+    this.stop = const Value.absent(),
+  });
+  TimersCompanion.insert({
+    this.id = const Value.absent(),
+    required DateTime start,
+    this.stop = const Value.absent(),
+  }) : start = Value(start);
+  static Insertable<Timer> custom({
+    Expression<int>? id,
+    Expression<DateTime>? start,
+    Expression<DateTime>? stop,
+  }) {
+    return RawValuesInsertable({
+      if (id != null) 'id': id,
+      if (start != null) 'start': start,
+      if (stop != null) 'stop': stop,
+    });
+  }
+
+  TimersCompanion copyWith(
+      {Value<int>? id, Value<DateTime>? start, Value<DateTime?>? stop}) {
+    return TimersCompanion(
+      id: id ?? this.id,
+      start: start ?? this.start,
+      stop: stop ?? this.stop,
+    );
+  }
+
+  @override
+  Map<String, Expression> toColumns(bool nullToAbsent) {
+    final map = <String, Expression>{};
+    if (id.present) {
+      map['id'] = Variable<int>(id.value);
+    }
+    if (start.present) {
+      map['start'] = Variable<DateTime>(start.value);
+    }
+    if (stop.present) {
+      map['stop'] = Variable<DateTime>(stop.value);
+    }
+    return map;
+  }
+
+  @override
+  String toString() {
+    return (StringBuffer('TimersCompanion(')
+          ..write('id: $id, ')
+          ..write('start: $start, ')
+          ..write('stop: $stop')
+          ..write(')'))
+        .toString();
+  }
+}
+
+class $TimersTable extends Timers with TableInfo<$TimersTable, Timer> {
+  @override
+  final GeneratedDatabase attachedDatabase;
+  final String? _alias;
+  $TimersTable(this.attachedDatabase, [this._alias]);
+  final VerificationMeta _idMeta = const VerificationMeta('id');
+  @override
+  late final GeneratedColumn<int> id = GeneratedColumn<int>(
+      'id', aliasedName, false,
+      type: DriftSqlType.int,
+      requiredDuringInsert: false,
+      defaultConstraints: 'PRIMARY KEY AUTOINCREMENT');
```

Comment:
I have the following error on this line:
>The argument type 'String' can't be assigned to the parameter type 'void Function(GenerationContext)?'.

If I delete this line the project compiles and run.
Does anyone else has this error?

Reply:
This file is generated when you run the "build_runner". I think it's documented on the readme how to generate. Have you tried generating it again and running the file?

Reply:
Ran `flutter pub run build_runner build --delete-conflicting-outputs` and the file is now updated and errors removed.
I needed to add the flag `--delete-conflicting-outputs` to refresh the file.
Should the `database.g.dart` file be in `.gitignore`?

Readme section where the command is described: https://github.com/dwyl/flutter-stopwatch-tutorial/tree/initial-implementation#persisting-timers

> Run flutter pub run build_runner build and you will notice a database.g.dart file was created. To use this file, change the MyDatabase class defined in the database.dart file defined earlier.